src/docker.go

```diff
@@ -68,13 +69,18 @@ func main() {
 	var fetcher raw.Fetcher
 	var docker nri.DockerClient
 	if args.Fargate {
-		metadataV3BaseURL, err := aws.MetadataV3BaseURL()
+		var err error
+		var metadataBaseURL *url.URL
+		if metadataBaseURL, err = aws.MetadataV4BaseURL(); err != nil {
+			//If we do not find V4 we fall back to V3
```

Comment:
i'm a bit worried about masking aws.MetadataV4BaseURL() errors here. 
perhaps adding a debug log here with printing the error could solve that, wdyt?

Reply:
you are right, I'll add a log